commit ab0d041b47afa70fb754121c020a92b107b6fb34 (refs/changes/73/4573/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:48:58+00:00 (1 year, 2 months ago)
    
    TRITON-508 config-agent autometadata should contain an array of IPs for each nic (bump version)

diff --git a/package.json b/package.json
index 5a00fc0..701079c 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "config-agent",
     "description": "SmartDataCenter Config Agent",
-    "version": "1.7.1",
+    "version": "1.8.0",
     "author": "Joyent (joyent.com)",
     "dependencies": {
         "assert-plus": "0.1.5",
